lib/controllers/combined_files.js

```diff
@@ -108,7 +108,7 @@ BT.CombinedFilesController = SC.ArrayController.extend({
     files.forEach(function (f) {
       ret.push("/* begin of file %@ */".fmt(f.get('path')));
       ret.push(f.get('content') || "");
-      ret.push(';');
+      if(f.isScript) ret.push(';');
```

Comment:
Even this won't really fix what you are trying to do. It still can cause spurious semicolons to appear.